Tasks/GoTool/gotool.ts

```diff
@@ -0,0 +1,120 @@
+///<reference path="../../definitions/node.d.ts" />
```

Comment:
don't use typings anymore

Reply:
Done
changes incorporated in https://github.com/Microsoft/vsts-tasks/pull/6307